_sources/Selection/Exercises.rst

```diff
@@ -435,7 +435,7 @@ Graded Lesson Assignment
 
 .. container:: full_width
 
-    A year is a **leap year** if it is divisible by 4, unless it is the first year of a century and it is not divisible by 400.
+    A year is a **leap year** if it is divisible by 4, however a century year cannot be a leap year unless it is divisible by 400.
```

Comment:
Wow the previous one was confusing! @ap2322 @bgschiller, are you sure this new wording keeps the original logic?  (ie: the original wording was so confusing that I can't tell) Haha -- apologies if this is your baby, @bgschiller-- I likely could not have done a better job myself.

Reply:
I think [this website](https://www.timeanddate.com/date/leapyear.html) has the most clear definition, and the parameters also match the examples and test code 
 
_In the Gregorian calendar three criteria must be taken into account to identify leap years:
The year can be evenly divided by 4;
If the year can be evenly divided by 100, it is NOT a leap year, unless;
The year is also evenly divisible by 400. Then it is a leap year._

I think the suggested wording change more closely aligns with both the test code and more clear definition split into 3 points.